### PR TITLE
HOT 1997 showing deceased person in person card

### PR DESCRIPTION
**Pull request by vikas199**
### Jira Story

- [Identifying Deceased Person in Screening HOT-1997](https://osi-cwds.atlassian.net/browse/HOT-1997)

## Description
This story shows the deceased label in both screening and snapshot page

## Tests
- [x] I have included unit tests 
- [x] I have included feature tests 
- [ ] I have included other tests 
- [ ] I have NOT included tests 

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by vikas199** [comment]
jenkins please retest this

**Comment by pokharelmanish** [comment]
retest this please

File: app/assets/stylesheets/_constants.scss [hunk 1]
```diff
@@ -71,6 +72,7 @@ $small-border-radius: 1px;
 $base-border-radius: 2px;
 $large-border-radius: 3px;
 $larger-border-radius: 4px;
+$border-radius-deceased: 30px;
```
Comment (NagaPradeepGajula-osi) [inline app/assets/stylesheets/_constants.scss]: nice use of constants. 💯

File: app/assets/stylesheets/typography.scss [hunk 2]
```diff
@@ -114,6 +114,15 @@ pre {
   width: 100%;
 }
 
+.information-flag-deceased {
```
[the remark this answers is not in the document]
Comment (NagaPradeepGajula-osi) [reply app/assets/stylesheets/typography.scss]: I agree with less domain-specific part. Earlier I was talking/appreciated more about using the constants but not towards being domain-specific.

[the remark this answers is not in the document]
Comment (wmitchell-cambria) [reply app/assets/stylesheets/typography.scss]: Yeah, +1. I was in a bit of a harsh mode, I think. Using the constants is great; it's just the naming.

File: app/assets/stylesheets/search.scss [hunk 3]
```diff
@@ -26,3 +26,8 @@
     background-color: $light-grey;
   }
 }
+
+.edit-remove {
```
[the remark this answers is not in the document]
Comment (vikas199) [reply app/assets/stylesheets/search.scss]: ok i will move this to shame_overrides

File: app/assets/stylesheets/typography.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (vikas199) [reply app/assets/stylesheets/typography.scss]: i will move information-flag-deceased styles to shame_overrides and change the name to information-flag-rounded

**Pull request merged**